python/cudf_polars/tests/experimental/test_join.py

```diff
@@ -13,11 +13,15 @@
 @pytest.mark.parametrize("how", ["inner", "left", "right", "full", "semi", "anti"])
 @pytest.mark.parametrize("reverse", [True, False])
 @pytest.mark.parametrize("max_rows_per_partition", [1, 5, 10, 15])
-def test_join(how, reverse, max_rows_per_partition):
+@pytest.mark.parametrize("bcast_join_limit", [1, 16])
+def test_join(how, reverse, max_rows_per_partition, bcast_join_limit):
     engine = pl.GPUEngine(
         raise_on_fail=True,
         executor="dask-experimental",
-        executor_options={"max_rows_per_partition": max_rows_per_partition},
+        executor_options={
+            "max_rows_per_partition": max_rows_per_partition,
+            "bcast_join_limit": bcast_join_limit,
```

Comment:
Can you think of any way to test that these configuration operations are actually getting used in the operation? Maybe there's something we can observe about the translated expression?